lxd/main_cluster.go

```diff
@@ -136,7 +137,16 @@ func (c *cmdClusterEdit) Run(cmd *cobra.Command, args []string) error {
 
 	var nodes []db.RaftNode
 	err = database.Transaction(func(tx *db.NodeTx) error {
-		var err error
+		config, err := node.ConfigLoad(tx)
+		if err != nil {
+			return err
+		}
+
+		clusterAddress := config.ClusterAddress()
+		if clusterAddress == "" {
+			return fmt.Errorf("LXD is not clustered - set up clustering first with 'lxc cluster enable <host>'")
```

Comment:
Maybe:

`LXD is not clustered (missing "cluster.https_address") - set up clustering first with "lxc cluster enable <host>"`

So its clear what we are checking?

Reply:
Done :+1: